plugins/huaweicloud/pom.xml

```diff
@@ -18,6 +18,19 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
+    <repositories>
+        <repository>
+            <id>HuaweiCloudSDK</id>
+            <url>https://repo.huaweicloud.com/repository/maven/huaweicloudsdk/</url>
```

Comment:
as far i see, we do need this internal repository. All huaweicloud SDKs must be available in the maven central already. 
Please check this in the flexibleengine pom as well.

Reply:
Removed the internal repository.